_episodes/03-monitor-the-model.md

```diff
@@ -322,15 +314,25 @@ history = model.fit(X_train, y_train,
 ~~~
 {: .language-python}
 
-We can plot the training process using the `history` object returned from the model training:
+We can plot the training process using the `history` object returned from the model training.
+We will create a function for it, because we will make use of this more often in this lesson!
 ~~~
 import seaborn as sns
 import matplotlib.pyplot as plt
 
-history_df = pd.DataFrame.from_dict(history.history)
-sns.lineplot(data=history_df['root_mean_squared_error'])
-plt.xlabel("epochs")
-plt.ylabel("RMSE")
+def plot_history(metrics):
+    """
+    Plot the training history
+
+    Args:
+        metrics: A list of metrics to plot
```

Comment:
Change it to `Metrics(str, list): Metric or a list of metrics to plot`

Reply:
Is that a convention for docstrings? Which style?

Reply:
google style https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html